public/scripts/main.js

```diff
@@ -62,38 +59,59 @@ function changeCursor(elem,type) {
 }
 
 function checkFood(item) {
-    return FOOD_LIST.includes(item);
+  return FOOD_LIST.includes(item);
 }
 
 function onHorseMouseDown(img) {
-    var element = img.parentElement;
-    var isFood = checkFood(element.classList[element.classList.length - 1]);
-    if (isFood) {
-        element.classList.add('active');
-    }
+  var element = img.parentElement;
+  var isFood = checkFood(element.classList[element.classList.length - 1]);
+  if (isFood) {
+    element.classList.add("active");
+  }
 }
 
 function onHorseMouseUp(img) {
-    img.parentElement.classList.remove('active');
+  img.parentElement.classList.remove("active");
 }
 
-document.querySelector('.dark-mode-btn').addEventListener('click', function () {
-    document.querySelector('body').classList.toggle('dark-mode');
+document.querySelector(".dark-mode-btn").addEventListener("click", function() {
+  document.querySelector("body").classList.toggle("dark-mode");
 });
 
 const makePoo = () => {
-    let rand = Math.round(Math.random() * (10000) + 20000);
-    setTimeout(function() {
-        makePoo();
-        const poop = document.getElementsByClassName('poop')[0];
-        const displayStyle = window.getComputedStyle(poop).getPropertyValue('display');
-        poop.style.display = displayStyle === 'none' ? 'inline' : displayStyle;
-    }, rand);
+  let rand = Math.round(Math.random() * 10000 + 20000);
+  setTimeout(function() {
+    makePoo();
+    const poop = document.getElementsByClassName("poop")[0];
+    const displayStyle = window
+      .getComputedStyle(poop)
+      .getPropertyValue("display");
+    poop.style.display = displayStyle === "none" ? "inline" : displayStyle;
+  }, rand);
 };
 
-const cleanUpPoo = (poop) => {
-    const cursorType = document.getElementsByClassName('horse')[0].className;
-    if (cursorType === 'horse pitchfork') {
-        poop.style.display = 'none';
-    }
+const cleanUpPoo = poop => {
+  const cursorType = document.getElementsByClassName("horse")[0].className;
+  if (cursorType === "horse pitchfork") {
+    poop.style.display = "none";
+  }
 };
+
+/*
+*   Clap counter
+=============================
+*   Author - @adsingh14
+*/
+
+let clap = 1;
+const clapBtn = document.getElementById("clapBtn"),
+clapCounter = document.querySelector('.clapCounter');
+
+clapBtn.addEventListener('click', function(){
+    clapCounter.textContent = clap ++;
+    setTimeout(function () {
```

Comment:
It'd be great if you remove whitespace between function & it's parenthesis. 
❌: `function ()`
✔️ : `function()`